### PR TITLE
Optional use of nodemon when using docker-chs-development

### PR DESCRIPTION
**Pull request by SteveHicksCH**
### JIRA link
n/a

### Change description
For node applications, when you run them in "dev mode" and do a change then this application will be updated by coping over the changed files and  restarting the docker container (see file `Tiltfile.dev` for details)

This is a relatively slow process and can be speeded up by creating a separate docker image that uses `nodemon` to restart the node application (within the docker container) when any files are changed as specified in the `nodemon.conf` file.

If you do not manually update the `docker-chs-development` project `Tiltfile` then the additional files will be ignored 

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: docs/use-nodemon-in-docker-chs-development-dev-mode.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## How to use this?
+
+Firstly set the application into 'dev mode'
+
+Then update in the `docker-chs-development` project the `Tiltfile`:
+
+``` shell
+
+'repositories/overseas-entities-web/dev.docker-compose.yaml'
```
Comment (JoshFen95) [inline docs/use-nodemon-in-docker-chs-development-dev-mode.md:26]: " add 'repositories/overseas-entities-web/dev.docker-compose.yaml' "

File: docs/use-nodemon-in-docker-chs-development-dev-mode.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# How to use `nodemon` when running app in docker-chs-development "dev mode"
+
+## What is this about?
+
+For node applications, when you run them in "dev mode" and do a change then this application will be updated by coping over the changed files and  restarting the docker container (see file `Tiltfile.dev` for details)
```
Comment (JoshFen95) [inline docs/use-nodemon-in-docker-chs-development-dev-mode.md:5]: coping = copying

changed files and  restarting = double space

File: docs/use-nodemon-in-docker-chs-development-dev-mode.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+This is a relatively slow process and can be speeded up by creating a separate docker image that uses `nodemon` to restart the node application (within the docker container) when any files are changed as specified in the `nodemon.conf` file.
```
Comment (JoshFen95) [inline docs/use-nodemon-in-docker-chs-development-dev-mode.md:7]: speeded = sped

File: docs/use-nodemon-in-docker-chs-development-dev-mode.md (hunk 1 above)
Comment (SteveHicksCH) [reply docs/use-nodemon-in-docker-chs-development-dev-mode.md]: Updated

File: docs/use-nodemon-in-docker-chs-development-dev-mode.md (hunk 2 above)
Comment (SteveHicksCH) [reply docs/use-nodemon-in-docker-chs-development-dev-mode.md]: Updated

Comment (SteveHicksCH) [reply docs/use-nodemon-in-docker-chs-development-dev-mode.md]: Updated

**Review by JoshFen95 — APPROVED** [review]

**Pull request merged**